spec/models/post_spec.rb

```diff
@@ -1,34 +1,44 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  it "is valid user and title" do
-    # Create user
-    user = FactoryGirl.create(:user)
-    # Create post
-    post = user.posts.new(
-      title: Faker::Lorem.sentence,
-      content: Faker::Lorem.paragraph 
-    )
-
-    expect(post).to be_valid
-  end
-
-  it "is invalid without title" do
-    user = FactoryGirl.create(:user)
-
-    post = user.posts.new(title: nil)
 
-    expect(post).to have(1).errors_on(:title)
+  describe "Create post with user available" do
+    before :each do
+      @user = FactoryGirl.create(:user)
+    end
+
```

Comment:
Before(:all), which ensures that the sample users are created once, before all the tests in the block. This is an optimization for speed.